src/Command/UninstallCommand.php

```diff
@@ -53,19 +68,30 @@ protected function configure()
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
-        $workshop = $input->getArgument('workshop');
+        $workshopName = $input->getArgument('workshop');
         $output->writeln('');
 
         try {
+            $workshop = $this->workshopRepository->getByName($workshopName);
+        } catch (WorkshopNotFoundException $e) {
+            $output->writeln(sprintf(' <error> No workshops found matching "%s" </error>', $workshopName));
+            return;
+        }
+
+        try {
+            $this->linker->unlink($workshop, $input->getOption('force'));
             $this->uninstaller->uninstallWorkshop($workshop);
         } catch (WorkshopNotInstalledException $e) {
-            $output->writeln(sprintf(' <error>Workshop "%s" not currently installed</error>', $workshop));
+            $output->writeln(sprintf(' <error> Workshop "%s" not currently installed </error>', $workshop->getName()));
             return;
         } catch (\RuntimeException $e) {
-            $output->writeln(sprintf(' <error>Failed to uninstall workshop "%s"</error>', $workshop));
+            $output->writeln([
```

Comment:
Maybe we should give a but more info on failures? Passing the exception message?